<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="turquoise" />
  <arg name="control_rate" default="10" />

  <group ns="$(arg namespace)">
    <node pkg="auv_mapping" type="object_map_tf_server_node" name="object_map_tf_server_node">
      <param name="rate" value="$(arg control_rate)" />
      <param name="static_frame" value="odom" />
      <remap from="set_object_transform" to="map/set_object_transform" />
    </node>

    <node pkg="auv_mapping" type="prop_transform_publisher.py" name="prop_transform_publisher"
      output="screen" />
    <node pkg="auv_mapping" type="gate_trajectory_publisher.py" name="gate_trajectory_publisher">
      <param name="gate_frame1" value="gate_blue_arrow_link" />
      <param name="gate_frame2" value="gate_red_arrow_link" />
      <param name="gate_exit_frame" value="gate_exit" />
      <param name="gate_enterance_frame" value="gate_enterance" />
      <param name="gate_selected_frame" value="gate_blue_arrow_link" />
      <param name="world_frame" value="odom" />
    </node>
  </group>
  <!--
  <node pkg="auv_control" type="bin_transform_publisher.py" name="bin_transform_publisher" />
  </include> -->
</launch>
